### ISSUE TITLE
Tests fail only with coverage

### ISSUE THREAD

**Issue by ilanDoron** [sc-forks/solidity-coverage#249]
Hi
I am using coverage tool for a version update.
About 32 tests fail only in coverage run. Out of 32 fails I can explain 2.
its this repo:
https://github.com/KyberNetwork/smart-contracts/tree/ilan/checkMaxDestOnProxy
Can u help me figure it out.
I am running with coverage tool 0.4.9

**Comment by cgewecke** [comment]
Yes - this is a known issue that especially affects wallet implementations. It's common if you have a wallet in your contract set to skip it for coverage run using the `skipFiles` option.
